[2023-12-07 22:09:33,172] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: ETL_redshift_docker_airflow.conexion_BD manual__2023-12-07T21:49:05.785692+00:00 [queued]>
[2023-12-07 22:09:33,184] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: ETL_redshift_docker_airflow.conexion_BD manual__2023-12-07T21:49:05.785692+00:00 [queued]>
[2023-12-07 22:09:33,185] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-12-07 22:09:33,185] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2023-12-07 22:09:33,185] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-12-07 22:09:33,199] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): conexion_BD> on 2023-12-07 21:49:05.785692+00:00
[2023-12-07 22:09:33,203] {standard_task_runner.py:52} INFO - Started process 539 to run task
[2023-12-07 22:09:33,206] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'ETL_redshift_docker_***', 'conexion_BD', 'manual__2023-12-07T21:49:05.785692+00:00', '--job-id', '161', '--raw', '--subdir', 'DAGS_FOLDER/ETL a Redshift.py', '--cfg-path', '/tmp/tmpoifme752', '--error-file', '/tmp/tmphbvyj4vg']
[2023-12-07 22:09:33,207] {standard_task_runner.py:80} INFO - Job 161: Subtask conexion_BD
[2023-12-07 22:09:33,266] {task_command.py:371} INFO - Running <TaskInstance: ETL_redshift_docker_airflow.conexion_BD manual__2023-12-07T21:49:05.785692+00:00 [running]> on host 4357a10b93bd
[2023-12-07 22:09:33,303] {logging_mixin.py:115} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:202 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2023-12-07 22:09:33,337] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=ManuelMoyano
AIRFLOW_CTX_DAG_ID=ETL_redshift_docker_***
AIRFLOW_CTX_TASK_ID=conexion_BD
AIRFLOW_CTX_EXECUTION_DATE=2023-12-07T21:49:05.785692+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-12-07T21:49:05.785692+00:00
[2023-12-07 22:09:33,339] {logging_mixin.py:115} INFO - Conectandose a la BD en la fecha: 2023-12-07 21
[2023-12-07 22:09:37,656] {logging_mixin.py:115} INFO - Connected to Redshift successfully!
[2023-12-07 22:09:37,656] {python.py:173} INFO - Done. Returned value was: None
[2023-12-07 22:09:37,673] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=ETL_redshift_docker_***, task_id=conexion_BD, execution_date=20231207T214905, start_date=20231207T220933, end_date=20231207T220937
[2023-12-07 22:09:37,715] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-12-07 22:09:37,752] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
